ts/components/screens/BaseHeader.tsx

```diff
@@ -271,10 +256,9 @@ class BaseHeaderComponent extends React.PureComponent<Props, State> {
         )}
 
         {/* if no right button has been added, add a hidden one in order to make the body always centered on screen */}
-        {!customRightIcon &&
-          !isSearchAvailable &&
-          !onShowHelp &&
-          !hasInstabugChat && <ButtonDefaultOpacity transparent={true} />}
+        {!customRightIcon && !isSearchAvailable && !onShowHelp && (
```

Comment:
```suggestion
       {!customRightIcon && !isSearchAvailable && !onShowHelp && !showChat && (
```
afik showChat is not used, at the moment
but for the future we should include it in this condition.
What do you think?